test_scripts/Defects/5_0/1862_SDL_does_not_validate_invalid_characters_in_HMI_requests.lua

```diff
@@ -0,0 +1,75 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1862
+--
+-- In case:
+-- 1) HMI sends SDL.GetUserFriendlyMessage request to SDL
+-- and this request has at least one String param with '\n' and/or '\t' and/or 'whitespace' as the only symbol(s).
+--
+-- Expected result:
+-- 1) SDL responds with 'INVALID_DATA' to HMI
+-- 2) Log corresponding error internally
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function testGetUserFriendlyMessageValidParams()
+  local rqIdValid = common.getHMIConnection():SendRequest("SDL.GetUserFriendlyMessage", {
+    language = "EN-US",
+    messageCodes = {"DataConsent"}
+  })
+  common.getHMIConnection():ExpectResponse(rqIdValid, {result = {code = 0, method = "SDL.GetUserFriendlyMessage"}})
+end
+
+local function testGetUserFriendlyMessageRequest(pRqParams)
+  local cid = common.getHMIConnection():SendRequest("SDL.GetUserFriendlyMessage", pRqParams)
+  common.getHMIConnection():ExpectResponse(cid, { error = {code = 11, data = {method = "SDL.GetUserFriendlyMessage" }}})
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+
+runner.Title("Test")
+runner.Step("GetUserFriendlyMessage valid params", testGetUserFriendlyMessageValidParams)
+runner.Step("GetUserFriendlyMessage madatory endline", testGetUserFriendlyMessageRequest, {
```

Comment:
typo madatory -> mandatory also in 45 and 50 lines

Reply:
@GetmanetsIrina Done in f6c6977